general/ai/cli.go

```diff
@@ -59,25 +64,75 @@ func HowCmd(c *cli.Context) error {
 			return err
 		}
 		log.Output(coreutils.PrintLink(llmAnswer))
+
+		err = sendFeedback()
+		if err != nil {
+			return err
+		}
+
 		log.Output("\n" + coreutils.PrintComment("-------------------") + "\n")
 	}
 }
 
+type QuestionBody struct {
+	Question string `json:"question"`
+}
+
 func askQuestion(question string) (response string, err error) {
-	contentBytes, err := json.Marshal(QuestionBody{Question: question})
+	return sendRestAPI(ask, QuestionBody{Question: question})
+}
+
+type FeedbackBody struct {
+	IsGoodResponse bool `json:"is_good_response"`
```

Comment:
`IsGoodResponse` can be made private.

Reply:
if it will be private it won't be sent to the server @eyalbe4